.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  pull_request: ~
+  push: ~
+
+jobs:
+  phpunit:
+    name: PHPUnit
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      matrix:
```

Comment:
IMO it would be nice to configure the strategy as `fail-fast: false` (see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast).
By doing this, if a job fails, the other jobs won't be stopped and will continue to run.

Reply:
Sure. Good idea.